contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -771,24 +769,20 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     ) private returns (TeleporterMessageReceipt[] memory result) {
         // Get the current outstanding receipts for the given chain ID.
         // If the queue contract doesn't exist, there are not outstanding receipts to send.
-        ReceiptQueue outstandingReceiptForDestination = outstandingReceipts[
-            chainID
-        ];
-        if (address(outstandingReceiptForDestination) == address(0)) {
+        uint256 outstandingReceiptsSize = outstandingReceipts[chainID].size();
```

Comment:
I think it will be cleanest to move `_getOutstandingReceiptsToSend` to now be a function in the `ReceiptQueue` library.

Reply:
move `getOutstandingReceiptsToSend` into the library.